### PR TITLE
AC Automated Test - LRAC-8626 | master

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LRAC-8626

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase [hunk 1]
```diff
@@ -65,9 +65,59 @@ definition {
 	@ignore = "true"
 	@priority = "5"
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase:66]: hi @epark18 , could you please remove the ignore? Thanks!

**Comment by alexcandidos** [comment]
Hi @epark18 , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**